app/views/layouts/application.html.erb

```diff
@@ -11,9 +11,41 @@
 
 </head>
 <body>
+  <!-- nav -->
+  <nav class="navbar navbar-default navbar-fixed-top">
```

Comment:
consider wrapping all the `<nav>` in it's own partial `layouts/_navbar.html.erb` and then including it with `<%= render 'layouts/navbar' %>`